filters/ignore_new_executors.hpp

```diff
@@ -0,0 +1,56 @@
+#include <ctime>
+#include <memory>
+
+#include "messages/serenity.hpp"
+
+#include "serenity/executor_map.hpp"
+#include "serenity/serenity.hpp"
+
+#include "stout/option.hpp"
+
+#ifndef SERENITY_IGNORE_NEW_TASKS_HPP
```

Comment:
I wondered if we are in 'Filters' directory should we duplicate the 'filter' everywhere?
If we are in this dir, that means that this class IS a filter.

Just for consideration.